backend/tests/Feature/Web/Admin/WorkOrderMaterialReconciliationTest.php

```
@@ -0,0 +1,123 @@
+        $this->actingAs($this->admin())
+            ->post($this->base()."/allocations/{$this->allocation->id}/return", ['qty' => 25])
+            ->assertRedirect()
+            ->assertSessionHas('success');
+
+        $this->assertEqualsWithDelta(425.0, (float) $this->material->fresh()->stock_quantity, 0.0001);
+        $this->assertEqualsWithDelta(75.0, (float) $this->allocation->fresh()->allocated_qty, 0.0001);
+    }
+
+    public function test_admin_reclassifies_to_another_class(): void
+    {
+        $target = Material::create([
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Material::create().


(staticMethod.notFound)